platform/android/src/native_map_view.cpp

```diff
@@ -391,6 +391,11 @@ void NativeMapView::createSurface(ANativeWindow *window_) {
     if (!firstTime) {
         firstTime = true;
 
+        oldDisplay = eglGetCurrentDisplay();
```

Comment:
Probably you no longer need to get the oldDisplay, surface, etc again on `::activate()`